portfolio/src/main/webapp/index.html

```diff
@@ -23,14 +23,26 @@ <h1>Tal's Portfolio</h1>
       <p>Click here to get a random fact about me!:</p>
       <button id = "button" onclick="addRandomFact()">About Me</button>
       <div id="fact-container"></div>
-      <p>Click here to see a random greeting:</p>
-      <button onclick="getRandomGreeting()">Greet Me!</button>
-      <div id="quote-container"></div>
-
+ 
+      <p>Post a comment here!</p>
       <form method="POST" action="/data">
-        <input type="text" name="title" />
-          <br/>
+        <input type="text" name="text" />
+        <br/>
+        <button id = "button">Submit Comment</button>
+
       </form>
+    
+      <p>How many comments should display:</p>
+      <form method="POST" action="/data">
+        <input type="number" name="choice" min="0" max="15" value="0">
```

Comment:
nit:  The servlet considers zero to be an invalid value (and would return 3 comments instead).  It probably shouldn't be an option here since you wouldn't want your users submitting invalid values to your server.  Not to mention your server is built to return something that's different from what the user is expecting (zero comments).  I would suggest adjusting this to only allow 1-15, or adjust the servlet to allow 0-15.